test/account.test.js

```diff
@@ -145,7 +145,7 @@ describe('with deploy contract', () => {
         expect(logs.length).toBe(3);
         expect(logs[0]).toEqual(`[${contractId}]: LOG: log before assert`);
         expect(logs[1]).toMatch(new RegExp(`^\\[${contractId}\\]: ABORT: "expected to fail" filename: "assembly/main.ts" line: \\d+ col: \\d+$`));
-        expect(logs[2]).toEqual(`[${contractId}]: Runtime error: wasm async call execution failed with error: WasmerCallError("Smart contract has explicitly invoked \`panic\`.")`);
+//        expect(logs[2]).toEqual(`[${contractId}]: Runtime error: wasm async call execution failed with error: WasmerCallError("Smart contract has explicitly invoked \`panic\`.")`);
```

Comment:
accidental?

Reply:
That's @mikhailOK change to support new error format in staging, until we move towards proper error handling.

Reply:
@evgenykuzyakov @mikhailOK probably good idea to add TODO with GH issue / WIP PR link so it doesn't get lost